packages/application-shell/src/utils/http-client.ts

```diff
@@ -42,9 +42,10 @@ export type TForwardToConfig = {
    */
   audiencePolicy?: TForwardToAudiencePolicy;
   /**
-   * The claims to be included in the exchange JWT sent to the external API.
+   * Request the Merchange Center API to include the user permission in the
+   * exchange token sent in the forwarded request to the external API.
```

Comment:
Nit (to follow the wording used for the other options)

```suggestion
   * A list of user permissions to be included in the request to the external API.
```

Reply:
Updated in this commit: de46f8f9da017cfa7be43230c0d08b17eaa67e0b